### PR TITLE
OHRM5X-2233: Add Deleted and Inactive tags for Claim Event/Expense Types

### PR DESCRIPTION
**Pull request by DulsaraNethmin**

### CODE REVIEW

File: installer/Migration/V5_5_0/lang-string/claim.yaml [hunk 1]
```diff
@@ -89,3 +89,15 @@ langStrings:
   -
     value: "Employee Claims"
     unitId: employee_claims
+  -
+    value: "Deleted Type"
```
Comment (ManulMax) [inline installer/Migration/V5_5_0/lang-string/claim.yaml:93]:
```suggestion
    value: "Deleted Expense Type"
```

File: src/client/src/orangehrmClaimPlugin/pages/submitClaim/SubmitClaim.vue [hunk 2]
```diff
@@ -40,7 +40,7 @@
               </oxd-grid-item>
               <oxd-grid-item>
                 <oxd-input-field
-                  v-model="claimEvent.name"
+                  v-model="formattedEventName"
```
Comment (ManulMax) [inline src/client/src/orangehrmClaimPlugin/pages/submitClaim/SubmitClaim.vue:43]: @Super-Chama can't we use placeholder here ?

File: installer/Migration/V5_5_0/lang-string/claim.yaml [hunk 3]
```diff
@@ -89,3 +89,15 @@ langStrings:
...
+    unitId: deleted_type
+  -
+    value: "Deleted Event"
+    unitId: deleted_event
+  -
+    value: "Inactive Type"
```
Comment (ManulMax) [inline installer/Migration/V5_5_0/lang-string/claim.yaml:99]:
```suggestion
    value: "Inactive Expense Type"
```

**Review by ManulMax — CHANGES_REQUESTED** [review]

File: src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/ClaimExpenseAPITestCases.yaml
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/ClaimExpenseAPITestCases.yaml:1]: I can't see any test cases for `status: false` and `isDeleted: true`

File: src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/ClaimRequestActionAPITestCases.yaml
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/ClaimRequestActionAPITestCases.yaml:1]: I can't see any test cases for `isDeleted: true`

File: src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/EmployeeClaimRequestAPITestCases.yaml
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/EmployeeClaimRequestAPITestCases.yaml:1]: I can't see any test cases for `isDeleted: true`

File: src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/MyClaimRequestAPITestCases.yaml
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/test/fixtures/testcases/MyClaimRequestAPITestCases.yaml:1]: I can't see any test cases for `isDeleted: true`

File: installer/Migration/V5_5_0/lang-string/claim.yaml [hunk 4]
```diff
@@ -89,3 +89,15 @@ langStrings:
...
+    unitId: deleted_expense_type
+  -
+    value: "Deleted Event"
+    unitId: deleted_event
+  -
+    value: "Inactive Expense Type"
+    unitId: inactive_expense_type
+  -
+    value: "Inactive Event"
+    unitId: inactive_event
```
Comment (RajithaKumara) [inline installer/Migration/V5_5_0/lang-string/claim.yaml:103]:
```yaml
  -
    value: "Deleted"
    unitId: deleted_expense_type
  -
    value: "Deleted"
    unitId: deleted_event
  -
    value: "Inactive"
    unitId: inactive_expense_type
  -
    value: "Inactive"
    unitId: inactive_event
```
double-check these values. Most probably these lang string already defined

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: src/client/src/orangehrmClaimPlugin/pages/submitClaim/SubmitClaim.vue (hunk 2 above)
Comment (Super-Chama) [reply src/client/src/orangehrmClaimPlugin/pages/submitClaim/SubmitClaim.vue:43]: can, what's the usecase here? claim event name should be pre populated?

**Comment by RajithaKumara** [comment]
Moved to #1705

**Pull request closed without merging**